src/mcts/params.cc

```diff
@@ -313,10 +313,10 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<ChoiceOption>(kHistoryFillId, history_fill_opt) = "fen_only";
   options->Add<FloatOption>(kMovesLeftMaxEffectId, 0.0f, 1.0f) = 0.0f;
   options->Add<FloatOption>(kMovesLeftThresholdId, 0.0f, 1.0f) = 1.0f;
-  options->Add<FloatOption>(kMovesLeftSlopeId, 0.0f, 1.0f) = 0.001f;
-  options->Add<FloatOption>(kMovesLeftConstantFactorId, -1.0f, 1.0f) = 1.0f;
+  options->Add<FloatOption>(kMovesLeftSlopeId, 0.0f, 1.0f) = 0.015f;
```

Comment:
I would prefer a more conservative value here like `0.005f`; the current TCEC shows that 0.007 with linear factor to be a bit too much for drawish positions around Q=0.4, and this value would reduce it to 1/3 there without influencing the effect on decided games too much.

Reply:
Sounds reasonable